### PR TITLE
Adds more med gear to R&D and Cargo

### PR DESCRIPTION
**Pull request by mistyLuminescence**
Hopefully we can make these an actual thing now.

- R&D can now create advanced (tier 2) health analyzers, and deconstruct them for tech levels. 
- Combat medical kits now have improved (tier 1) health analyzers.

Also, typo in defibril**l**ator crate fixed, new descriptions and origin tech added for t2/t3 health analyzers. Note that t3/phasic health analyzers are adminspawn-only, per request.

What does this mean?
- R&D can now supply medical with advanced health analyzers. R&D, medical loves you..)
- Mercs and antags have an easier time diagnosing more serious injuries when they buy combat medkits.

Edit: prototype medical gear no longer available through Cargo. Sorry, folks.

### CODE REVIEW

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -267,20 +267,24 @@ REAGENT SCANNER
...
 	advscan = 2
+	origin_tech = list(TECH_MAGNET = 6, TECH_BIO = 7)
 	icon_state = "advhealth"
 
 /obj/item/device/healthanalyzer/enhanced //reports all of the above, as well as name and quantity of nonmed reagents in stomach
 	name = "phasic health analyzer"
+	desc = "Possibly the most advanced health analyzer to ever have existed, utilising bluespace technology to determine almost everything worth knowing about a patient."
 	advscan = 3
+	origin_tech = list(TECH_MAGNET = 7, TECH_BIO = 8)
 	icon_state = "advhealth"
```
Comment (Arokha) [inline code/game/objects/items/devices/scanners.dm]: I'm a little saddened by the fact that all 3 have the same iconstate.

File: code/game/objects/items/devices/scanners.dm [hunk 2]
```diff
@@ -267,20 +267,24 @@ REAGENT SCANNER
 			to_chat(usr, "The scanner will now perform a basic analysis.")
 
 /obj/item/device/healthanalyzer/improved //reports bone fractures, IB, quantity of beneficial reagents in stomach; also regular health analyzer stuff
-	name = "advanced health analyzer"
+	name = "enhanced health analyzer"
```
Comment (Arokha) [inline code/game/objects/items/devices/scanners.dm]: This one is named "enhanced", but the one with "enhanced" in the path is "phasic"...?

Comment (mistyLuminescence) [reply code/game/objects/items/devices/scanners.dm]: good spot

File: code/game/objects/items/devices/scanners.dm (hunk 1 above)
Comment (mistyLuminescence) [reply code/game/objects/items/devices/scanners.dm]: >implying sprites
if we can get a colorscheme I can make sprites

Comment (mistyLuminescence) [reply code/game/objects/items/devices/scanners.dm]: update: sprites made, http://prntscr.com/i9fptd

File: code/datums/supplypacks/medical.dm [hunk 3]
```diff
@@ -330,8 +330,18 @@
...
-	name = "Defibrilator crate"
+	name = "Defibrillator crate"
 	contains = list(/obj/item/device/defib_kit = 2)
 	cost = 30
 	containertype = /obj/structure/closet/crate/medical
-	containername = "Defibrilator crate"
\ No newline at end of file
+	containername = "Defibrillator crate"
+
+/datum/supply_packs/med/advgear
```
[the remark this answers is not in the document]
Comment (mistyLuminescence) [reply code/datums/supplypacks/medical.dm]: 👍 done

**Pull request merged**